### PR TITLE
Read me

### LINKED ISSUE #1: Readme
### Writing Readme file 
You should include

- the aim of the project.
- the team members.
- description of the project and its tasks.
- the distribution of project tasks to the team members.
- the reviewers of each task.

### PR DESCRIPTION
**Pull request by Ola014**

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -0,0 +1,172 @@
+*{
+    padding: 0;
+    margin: 0;
```
Comment (sara219) [inline index.css]: pull the main and merge it into your branch (locally) 
then remove everything not related to README.md

File: README.md [hunk 2]
```diff
@@ -1,2 +1,31 @@
...
+
+The aim of this project is build a website selling domestic furniture and offering interior design services. The website will contain two pages which are the home page and the about page. 
+## member of the team
+
+People working in this project are:
+- Abed Al-Rahman Hassan
+- Ahmed Hamouda
+- Ibrahim Jarada
+- Ola A. Ahil
+ The leader for this project is Sara J, our dear mentor.
```
Comment (sara219) [inline README.md]: you don't have to add my name there, remove it pls :)

File: README.md [hunk 3]
```diff
@@ -1,2 +1,31 @@
 # E-commerce-group2-project
```
Comment (sara219) [inline README.md]: Retype the name, perhaps in another way without the dashes

_ex:_ 
> E-commerce | Athathy

**Review by sara219 — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (Ola014) [reply README.md]: Ok.

**Review by sara219 — CHANGES_REQUESTED** [review]
Resolve the conflict

**Review by Ibrahim-Jarada — CHANGES_REQUESTED** [review]
take a copy from the lasted readme file in a backup folder
open E-commerce-group2-project folder => git bash here
git checkout read-me
git pull origin read-me
copy readme file from the backup folder
paste it to E-commerce-group2-project folder replacing the existing one
git add .
git commit -m " Readme file
Relates #1"
git push origin read-me

**Review by Ibrahim-Jarada — APPROVED** [review]

**Review by sara219 — APPROVED** [review]

**Review by ahmedhamouda2 — APPROVED** [review]

**Pull request merged**